### PR TITLE
Added AfterPay countries available.

### PR DESCRIPTION
**Pull request by LeasoftKonstantinShapoval**

### CODE REVIEW

File: emspay.php [hunk 1]
```diff
@@ -285,6 +285,37 @@ function afterpay_filter_gateways($gateways)
...
+     * @return mixed
+     */
+    function afterpay_countries_available($gateways)
+    {
+        $settings = get_option('woocommerce_emspay_afterpay_settings');
+        $ap_countries_available = $settings['ap_countries_available'];
+
+        if (strlen($ap_countries_available) > 0) {
+            $countrylist = explode(",", str_replace(' ', '', $ap_countries_available));
+            if (!in_array(WC()->customer->billing['country'], $countrylist)) {
```
Comment (tiutiunnykleasoft) [inline emspay.php:303]: изображение

**Review by tiutiunnykleasoft — CHANGES_REQUESTED** [review]
Error on the checkout page

File: emspay.php (hunk 1 above)
Comment (LeasoftKonstantinShapoval) [reply emspay.php:303]: This errors is not for my side. Check server settings, all install requires and follow install order. Checked by new clear WP site.

Comment (tiutiunnykleasoft) [reply emspay.php:303]: @LeasoftKonstantinShapoval  : What version wooCommerce you are using ?

**Review by tiutiunnykleasoft — APPROVED** [review]
Ok.

**Pull request merged**